packages/optimizer/spec/transformers/experimental/AmpStoryCssTransformer/config.json

```diff
@@ -0,0 +1,3 @@
+{
+  "optimizeAmpStory": true
```

Comment:
@sebastianbenz am i able to create config.json inside an individual test folder. I wanted to test one scenario for LTS

Reply:
For individual test file you can add an inline comment like here: https://github.com/ampproject/amp-toolbox/blob/main/packages/optimizer/spec/transformers/valid/RewriteAmpUrls/adds_lts/input.html#L1